client/sections-middleware.js

```diff
@@ -1,24 +1,18 @@
-/**
- * External dependencies
- */
-import page from 'page';
-
-/**
- * Internal dependencies
- */
 import config from '@automattic/calypso-config';
+import * as LoadingError from 'calypso/layout/error';
+import { performanceTrackerStart } from 'calypso/lib/performance-tracking';
+import { addReducerToStore } from 'calypso/state/add-reducer';
+import { bumpStat } from 'calypso/state/analytics/actions';
 import { setSectionLoading } from 'calypso/state/ui/actions';
 import { activateNextLayoutFocus } from 'calypso/state/ui/layout-focus/actions';
-import { bumpStat } from 'calypso/state/analytics/actions';
-import * as LoadingError from 'calypso/layout/error';
+import page from 'page';
```

Comment:
Maybe we should use [`import/internal-regex`](https://github.com/benmosher/eslint-plugin-import/blob/master/README.md#importinternal-regex) so specify that `calypso` is an internal module and should be in its own group.

Reply:
Agreed. Should we open a separate PR or include it in this one?

Reply:
Let's do it in this one so we can see the effect in a small file

Reply:
Added here https://github.com/Automattic/wp-calypso/pull/54494/commits/adc56a109ec18040a82d78693aff9e91cb713380 – intuitively I'd group `^calypso/` packages before packages with relative path but that's a very very minor nit.

Reply:
I think it can be done with https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array. But minor nit indeed, I don't have a strong opinion.